tests/framework/extensions/provisioning/ssh.go

```diff
@@ -81,6 +95,35 @@ func CallSSHTestByName(testCase provisioninginput.SSHTestCase, node *nodes.Node,
 		}
 
 		return err
+
+	//This test checks if the audit log file is properly created on the node (skipped if its not a control plane node).
+	//For k3s you will need to configure the audit log dir to: /var/lib/rancher/k3s/etc/config-files/audit-policy-file
+	case auditLog:
+		mgmtcluster, err := client.Management.Cluster.ByID(clusterID)
```

Comment:
Can you change this to `cluster` and not `mgmtcluster`? We typically follow that naming practice.

Reply:
@markusewalker @slickwarren are you guys ok with the norman API call to do this? I think we should limit new norman calls moving forward. Although in this case it's not a big deal ...

Reply:
As you said we shouldn't be using Norman, I'll look around for a more up to date way to do this.

Reply:
You could do something like this, we do thins snapshot restore https://github.com/rancher/rancher/blob/d8a077eb7f1f25ffa66e5e4ba91bc29eb7c85cd4/tests/v2/validation/snapshot/snapshot.go#L269C4-L282